src/Http/ActionDispatcher.php

```diff
@@ -64,16 +64,19 @@ public function __construct($factory = null, $eventManager = null, array $filter
             $this->addFilter($filter);
         }
         $this->factory = $factory ?: new ControllerFactory();
+
+        // Force aliases to be autoloaded.
+        class_exists('Cake\Network\Request');
```

Comment:
@ADmad I think that will just force the `Request.php` with the `class_alias` statement to be loaded. It's lightweight but I'm not sure why he's doing it. I guess it fixes something.